molgenis-data-elasticsearch/pom.xml

```diff
@@ -55,5 +55,10 @@
             <artifactId>protonpack</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.molgenis</groupId>
+            <artifactId>molgenis-mail</artifactId>
```

Comment:
The mail module will be a very small Maven module. Maybe this is the time to introduce molgenis-utils and add the mail package there? Afterward we can migrate and ditch molgenis-core.